databases/week1/datab.sql

```diff
@@ -0,0 +1,29 @@
+ALTER TABLE
+	status CHANGE id id int unsigned NOT NULL;
+	select COUNT(*)from task;
+	--Find out how many tasks in the task table do not have a valid due date
+	SELECT * from task where due_date is NULL; 
```

Comment:
The small modification. See if it makes more sense...
SELECT count(title) from task where due_date is NULL;